content/rc/databases/view-edit-database.md

```diff
@@ -160,21 +150,27 @@ The **Databases** list summarizes the status of all databases from the subscript
 
 You can:
 
-- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include **Subscription**, **Name**, and **Memory**.
+- Search by typing into the search box located above the database list.
 
-    {{<image filename="images/rc/icon-database-list-sort-ascending.png" alt="Use the arrows in the list header to sort the list." >}}{{< /image >}} {{<image filename="images/rc/icon-database-list-sort-descending.png" alt="The direction of the arrow corresponds to the direction of the sort." >}}{{< /image >}}
+    {{<image filename="images/rc/database-list-search.png" alt="Use the search bar to filter the list." >}}{{< /image >}}
 
-    Select the arrow icon to change the sort order.  One sort order can be active at any given time.
+- Filter by selecting a filter type and then selecting the checkbox next to the options you want to include from the dropdown.  Select the Filter toggle, located on the right of the search bar, if the filter types are hidden.
```

Comment:
Maybe the filter section paragraphs should be rearranged slightly. I think it makes more sense to start with "Filter the list on Status, Subscription, Capabilities, and Options" first, and then explain how to display and select the filter options.